src/ui/patchSettings.tsx

```diff
@@ -54,28 +47,28 @@ export default function patchSettings() {
                         label="Aliucord"
                         trailing={FormRow.Arrow}
                         onPress={() =>
-                            navigation.push("ASettings")
+                            undefined//navigation.push("ASettings")
```

Comment:
doesn't this just make settings non functional

Reply:
yes but the navigation doesnt exist